### ISSUE TITLE
add gageID/comid to plot_output filename

### ISSUE THREAD

**Issue by ryanpeek** [ceff-tech/ffc_api_client#19]
Add some code to add the gageID to the filename for plot outputs. Currently it defaults to a metric/component and then "_" but nothing else (i.e., "DS_.png", "SP_.png"). Would help when dealing with multiple gages at once or iterating. I can mess with this later.

**Comment by nickrsan** [comment]
Good idea - I think I'd even expand it to the plotting code taking a general name parameter that would end up being something like "gage_1111111" or "comid_111111111" since it could be triggered from code using either a gage or a DF+COMID. Then we could have whatever calling function pass that name through from whichever functions initiate it.

**Comment by ryanpeek** [comment]
I like it...definitely would be a good approach and I like the consistency and availability of that naming convention across lots of things.

**Comment by nickrsan** [comment]
The COMID is now on the plots, but the gage ID isn't yet - should be a simple bit of logic to add, I think

**Comment by nickrsan** [comment]
I added the Gage ID - I'm going to leave this open so that in the future we can implement the "name" approach
